sqlglot/parser.py

```diff
@@ -1362,9 +1362,12 @@ def extend_props(temp_props: t.Optional[exp.Properties]) -> None:
             # exp.Properties.Location.POST_SCHEMA ("schema" here is the UDF's type signature)
             extend_props(self._parse_properties())
 
-            self._match(TokenType.ALIAS)
+            alias_ = self._match(TokenType.ALIAS)
```

Comment:
you can just put this inline in the if statement since you don't use alias_ anywhere

Reply:
I actually suggested we don't put that in the if haha.. the reasoning was to make this a bit more clear, i.e. that we'll always consume the alias regardless of the taken branch

Reply:
But it's fine